src/java/za/co/mip/ablduck/utilities/HTMLGenerator.java

```diff
@@ -0,0 +1,376 @@
+            deps = "<h4>Subclasses</h4>";
+
+            for (String subclass : cls.subclasses) {
+                deps += "<div class='dependency'>" + renderLink(subclass) + "</div>";
+            }
+        }
+        return deps;
+    }
+    
+    private String renderClassComment (SourceJSObject cls) {
+        String classComment = "";
+        
+        if (cls.meta.internal != null) 
+            classComment = renderInternal(cls.meta.internal) + "</br>";
+                
+        if (cls.meta.deprecated != null)
+            classComment = renderDeprecated("class", cls.meta.deprecated) + "</br>" + classComment;
+               
+        classComment += cls.comment;
+        return classComment;
+    }
+    
+    private String renderMemberDetails (SourceJSObject cls, String memberSection, String memberSectionTitle) {
+        
+        String member = "<div class='members-section'>"
+                          + "<div class='definedBy'>Defined By</div>"
+                          + "<h3 class='members-title icon-" + memberSection + "'>" + memberSectionTitle + "</h3>"
+                          + "<div class='subsection'>"
+                              + renderMember(cls, memberSection)
+                          + "</div>"
+                      + "</div>";
+        
+        return member;
+    }
+    
+    private String renderMember (SourceJSObject cls, String memberType) {
+        
+        String memberHTML = "";
+        Boolean first = true;
+        
+        for (MemberObject member : cls.members) {
+            
+            if (member.tagname.equals(memberType)) {
+
+                String firstChild = "";
+                if (first) {
+                    firstChild = "first-child";
+                    first = false;
+                }
+                
+                String inherited;
+                if (member.owner.equals(cls.name))
+                  inherited = "not-inherited";
+                else
+                  inherited = "inherited";
+                
+                String doc = "";
+                if(member.comment != null)
+                    doc = member.comment;
+                
+                String shortdoc = stripHtmlTags(doc);
+                if (shortdoc.length() > 100)
+                    shortdoc = shortdoc.substring(0, 100) + " ..."; 
+                
+                if(shortdoc.length() == 0) {
+                    shortdoc = "&nbsp;";
+                    doc      = "&nbsp;";
+                }
+                
+                if (member.meta.internal != null) 
+                    doc = renderInternal(member.meta.internal) + "</br>" + doc;
+                
+                if (member.meta.deprecated != null)
+                    doc = renderDeprecated(member.tagname, member.meta.deprecated) + "</br>" + doc;
+               
+                String sig = "";
+                String returnT = "";
+                
+                if (member.tagname.equals("method")) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move the "method" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1132)